samples/msgext-unfurling-ac-loop-components/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,74 @@
+{
+    "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
+    "manifestVersion": "1.16",
+    "version": "1.0.0",
+    "id": "${{TEAMS_APP_ID}}",
+    "packageName": "com.microsoft.teams.extension",
+    "developer": {
+        "name": "Teams App, Inc.",
+        "websiteUrl": "https://www.example.com",
+        "privacyUrl": "https://www.example.com/termofuse",
+        "termsOfUseUrl": "https://www.example.com/privacy"
+    },
+    "icons": {
+        "color": "color.png",
+        "outline": "outline.png"
+    },
+    "name": {
+        "short": "AC Loop Component",
+        "full": "full name for Msgext Unfurling Ac Loop Components"
+    },
+    "description": {
+        "short": "short description of Msgext Unfurling Adaptive Card Based Loop Components.",
+        "full": "full description of Messaging Extension Unfurling Adaptive Card Based Loop Components."
```

Comment:
This sample shows messaging extension with adaptive card-based loop components functionality.

Reply:
Fixed